charts/stigatron/Chart.yaml

```
@@ -9,3 +9,7 @@ dependencies:
     version: "0.1.41"
     repository: "https://rancherfederal.github.io/carbide-charts"
     alias: heimdall2
+  - name: nats
+    version: 1.1.4
```

```suggestion
    version: "1.1.4"
```

force this to be a string as is done with the heimdal2 dep